app/livechat/imports/server/rest/sms.js

```diff
@@ -1,11 +1,26 @@
+import { HTTP } from 'meteor/http';
 import { Meteor } from 'meteor/meteor';
 import { Random } from 'meteor/random';
 
+import { FileUpload } from '../../../../file-upload';
```

Comment:
Please always import at least from the `server` or `client` folders.. If possible, update other not "compliant" imports as well.

```suggestion
import { FileUpload } from '../../../../file-upload/server';
```